### PR TITLE
Add TLV75801P

### PR DESCRIPTION
**Pull request by adamheinrich**
Add TLV75801P in DBV and DRV (SOT-23 and WSON-6) packages.

Datasheet: https://www.ti.com/lit/ds/symlink/tlv758p.pdf

This PR intentionally violates S3.1 (Symbol slightly off-center). I based it on other TLV7x symbols which violate it too.

tlv75801pdbv
tlv75801pdrv

### CODE REVIEW

File: Regulator_Linear.dcm [hunk 1]
```diff
@@ -6462,6 +6462,18 @@ K LDO Regulator Fixed Positive
 F https://www.ti.com/lit/ds/symlink/tlv757p.pdf
 $ENDCMP
 #
+$CMP TLV75801PDBV
+D 500mA Low-Dropout Linear Regulator, Adjustable Output, SOT-23-5
+K linear regulator ldo adjustable positive
```
Comment (chschlue) [inline Regulator_Linear.dcm]: Remove redundant keywords, i.e. linear, regulator and adjustable.

File: Regulator_Linear.dcm [hunk 2]
```diff
@@ -6462,6 +6462,18 @@ K LDO Regulator Fixed Positive
...
+F https://www.ti.com/lit/ds/symlink/tlv758p.pdf
+$ENDCMP
+#
+$CMP TLV75801PDRV
+D 500mA Low-Dropout Linear Regulator, Adjustable Output, WSON-6
+K linear regulator ldo adjustable positive
```
Comment (chschlue) [inline Regulator_Linear.dcm]: See above

File: Regulator_Linear.lib [hunk 3]
```diff
@@ -4148,6 +4148,48 @@ X GND 7 0 -300 100 U 50 50 1 1 P N
...
+DEF TLV75801PDBV U 0 10 Y Y 1 F N
+F0 "U" -150 225 50 H V C CNN
+F1 "TLV75801PDBV" 0 225 50 H V L CNN
+F2 "Package_TO_SOT_SMD:SOT-23-5" 0 325 50 H I C CIN
+F3 "" 0 50 50 H I C CNN
+$FPLIST
+ SOT?23*
+$ENDFPLIST
+DRAW
+S -200 175 200 -200 0 1 10 f
```
Comment (chschlue) [inline Regulator_Linear.lib]: Move top edge to 200, no point in this asymmetry.

File: Regulator_Linear.lib [hunk 4]
```diff
@@ -4148,6 +4148,48 @@ X GND 7 0 -300 100 U 50 50 1 1 P N
...
+X IN 1 -300 100 100 R 50 50 1 1 W
+X GND 2 0 -300 100 U 50 50 1 1 W
+X EN 3 -300 0 100 R 50 50 1 1 I
+X FB 4 300 0 100 L 50 50 1 1 I
```
Comment (chschlue) [inline Regulator_Linear.lib]: Move down by 100 mils so an R_Small fits between OUT and FB

**Comment by aewallin** [comment]
NC pin placement should be on the box outline, like so https://kicad-pcb.org/img/klc/S4.6.png

File: Regulator_Linear.lib [hunk 5]
```diff
@@ -4148,6 +4148,48 @@ X GND 7 0 -300 100 U 50 50 1 1 P N
...
+$FPLIST
+ WSON*1EP*2x2mm*P0.65mm*
+$ENDFPLIST
+DRAW
+S -200 175 200 -200 0 1 10 f
+X OUT 1 300 100 100 L 50 50 1 1 w
+X FB 2 300 0 100 L 50 50 1 1 I
+X GND 3 0 -300 100 U 50 50 1 1 W
+X EN 4 -300 0 100 R 50 50 1 1 I
+X DNC 5 300 -100 100 L 50 50 1 1 N N
```
Comment (chschlue) [inline Regulator_Linear.lib]: Swap NC and FB pins, see above.

**Comment by adamheinrich** [comment]
@chschlue @aewallin resolved.

**Comment by chschlue** [comment]
Please don't force-push after a review.
Makes it hard to track actual changes.

**Pull request merged**